scale_test/dnspolicy/README.md

```diff
@@ -0,0 +1,144 @@
+# DNSPolicy Scale Testing
+
+Scale testing using [kube-burner](https://kube-burner.github.io/kube-burner/latest).
+
+## Setup local environment (kind)
+
+Create a kind cluster with kuadrant and prometheus/thanos installed and configured using the [kuadrant-operator](https://github.com/Kuadrant/kuadrant-operator) repo tasks and config.
+
+In the kuadrant-operator directory, run the following to create a local kind cluster with kuadrant deployed:
+```shell
+make local-setup SUBNET_OFFSET=0 CIDR=26 NUM_IPS=64
+```
+Note: In order to test at scale, metallb must be configured with enough ip address to assign one to each gateway to be created.  
+
+## Pre-test setup
+
+Scale down the default dns operator:
+```shell
+kubectl scale deployment/dns-operator-controller-manager --replicas=0 -n kuadrant-system
+```
+Note: This is required for the default kubeburner workload (namespaced-dns-operator-deployments.yaml) as it creates its own dns operator deployments.
+
+Deploy the observability stack:
+```shell
+#kubectl apply --server-side -k github.com/mikenairn/dns-operator/config/observability?ref=add_scale_test
+kubectl apply --server-side -k github.com/kuadrant/dns-operator/config/observability?ref=main # Run twice if it fails the first time
```

Comment:
This needs to be updated.

I think a single kustomization in the kuadrant operator that can be executed, without the need to pull down the repo, would be useful and easier than having to run this set of steps https://github.com/Kuadrant/kuadrant-operator/tree/main/config/observability#deploying-the-observabilty-stack. Depends how varied the observability setup gets.

Thanos setup should likely be a secondary optional task when working with a single cluster.  

@david-martin Any opinions/thoughts on this?

Reply:
The cmd you have here should be enough to deploy the stack (without thanos or example alerts & dashboards).
You would typically need to run it twice if CRDs don't exist.
Once to just include CRDs so they are registered, and a 2nd time (without CRDs) to avoid errors creating CRs.